tests/CMakeLists.txt

```diff
@@ -1,15 +1,17 @@
+cmake_policy(SET CMP0002 OLD) #Allow duplicate targets by overriding, needed for multiple tests that work with the same target
```

Comment:
I don't understand, what targets are getting duplicated? Does this problem go away if the tests are run under `ctest`?

Reply:
Some of the tests work with targets that use the same name, this was just a quick and dirty way to fix a conflict issue and will be removed as soon as I rename the targets used in each test